### PR TITLE
Make configuration compliant with latest symfony versions

### PR DESCRIPTION
**Pull request by msphn**
This change is probably breaking.
I could need your help to check if it's still working with 3.4 and 4!

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -34,8 +34,8 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder(): TreeBuilder
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('linkin_swagger_resolver');
+        $treeBuilder = new TreeBuilder('linkin_swagger_resolver');
+        $rootNode = $treeBuilder->getRootNode();
```
Comment (adrenalinkin) [inline DependencyInjection/Configuration.php]: Please use this construct
```php
        if (method_exists($treeBuilder, 'getRootNode')) {
            $rootNode = $treeBuilder->getRootNode();
        } else {
            // BC layer for symfony/config 4.1 and older
            $rootNode = $treeBuilder->root('linkin_swagger_resolver');
        }
```
And do not forget import `method_exists` into `use` section

**Review by adrenalinkin — CHANGES_REQUESTED** [review]

**Comment by msphn** [comment]
For me works fine now. :-)

**Comment by adrenalinkin** [comment]
@msphn Good job.
I plan add tests and release first major version. So keep in touch.

**Comment by msphn** [comment]
I can help you out, I set up CI before. You want to set up travis?

**Comment by adrenalinkin** [comment]
@msphn yes you can :)
But keep in mind - currently I have a lot of work on my job and can answer to you with delay

**Pull request merged**